action.yml

```diff
@@ -1,15 +1,6 @@
-name: '"AWS CodeBuild run build" Action For GitHub Actions'
-description: 'Execute CodeBuild::startBuild for the current repo.'
-branding:
-  icon: 'cloud'
-  color: 'orange'
+name: 'Easy generalized deployments'
+description: 'Kick off an easy generalized deployment'
 inputs:
-  project-name:
-    description: 'AWS CodeBuild Project Name'
-    required: true
-  buildspec-override:
-    description: 'Buildspec Override'
-    required: false
```

Comment:
We can remove project-name and buildspec-override since those will be hardcoded in the Lambda function